lua/solarized-osaka/theme.lua

```diff
@@ -313,9 +313,26 @@ function M.setup()
     ["@module"] = { link = "Include" },
 
     -- tsx
-    ["@tag.tsx"] = { fg = c.green500 },
+    -- ["@tag.tsx"] = { fg = c.green500 },
```

Comment:
Can you update this line instead of adding L319?

Reply:
I do think that instead of using `c.orange500` for the `@tag.tsx` it would be better to use `c.yellow500` as that would not blend in with the `<>` tags being used as they also have the `c.orange500` and they blend together too much when using Imported components together.
![Screenshot from 2024-03-06 23-09-35](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/4236ce3d-f2d2-4e6e-9a63-677694e324fb)

They blend together a little too much but using `c.yellow500` it looks better for differentiation and also is consistent with their import color on top.
![Screenshot from 2024-03-06 23-10-34](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/168b4288-ef7a-420a-84ac-0680a9f1d7b4)

This way all the colors are different for proper identification and also coherent in their hierarchy.
I will wait for your response and then adjust this accordingly if you still want the `@tag.tsx` to be `c.green500`.

Reply:
Sorry, I mean please change the L316 to use `orange500` because I don't like commenting out lines.
Yeah, `yellow500` looks better! Thanks

Reply:
I have removed the line and changed it to `yellow500`.